### ISSUE TITLE
[on hold] Refactor tab paragraph container component using new CMS component (not available)

### ISSUE THREAD

**Issue by tobi-or-not-tobi** [SAP/spartacus#1484]
Currently the tab system on the PDP is hardcoded. The inner components cannot easily be changed by customers, not at all by business users. In order to fix this, we like to refactor and leverage the `CMSTabParagraphContainer`.

The intended structure for the tab component:
- the wrapping container component is driven by CMS
- the container component references 4 components in a certain order (which can be altered by business users)
- the individual components can be re-configured by customers in the standard way (comp mapping)
- no need to have special outlets for tab component as well as the inner sections (i.e. shipping)

1. evaluate current CMS structure for the paragraph tab component (do we have the right structure in place)
2. evaluate tab header (not sure who this works)
3. (optional) create proper CMS structure
4. implement sub components (paragraph, specs, reviews, shipping)
5. implement / refactor container component
6. refactor overall css

**Comment by Xymmer** [comment]
depends on CMSX-8493 ?

**Comment by tobi-or-not-tobi** [comment]
@Pucek9 not sure if you've started this ticket, but i've done a bit of research as this becomes a priority for the folks who implement Yotpo integration. The cms structure we have today isn't going to help us enough, I've spotted these issues:
- The `CMSTabParagraphContainer` is currently configured for the `TabsSlot` on the PDP and holds only the `deliveryTab` which is a `CMSTabParagraphComponent`. Other components are supposed to be hardcoded. 
- The `CMSTabParagraphContainer` is semantically wrong, as we're supposed to nest specs, reviews, etc.
- While we can nest more components into the `CMSTabParagraphContainer`, we need to have a mechanism to provide a tab-title. There's no obvious structure that allows for this. The  `CMSTabParagraphComponent` provides a localised title for this reason, but it's only available for the paragraph component. We should not extend every component type in order to add a title. 

The suggested approach isn't going to work here (my bad). Let's put this ticket ON HOLD for now, I'll talk to the CMS team to see what we can do on short terms. Most likely we need a temporary solution here.

**Comment by tobi-or-not-tobi** [comment]
Proposed datamodel:

![image](https://user-images.githubusercontent.com/4745807/53895380-edc79080-3fff-11e9-9529-d4331b5828c9.png)

**Comment by Xymmer** [comment]
looks like we hve to wait until backend is updated

**Comment by cayacdev** [comment]
Hi together, maybe my Stackoverflow Question is related: https://stackoverflow.com/q/63371797/7365529 ?
